src/Phoenix/PhoenixClient.ts

```diff
@@ -130,7 +181,7 @@ export class PhoenixClient {
   }
 
   public getPhoenixContainerPoolingEndPoint(): string {
```

Comment:
Remove AccountData type, you can get the path using 
`${this.getPhoenixContainerPoolingEndPoint()}{userContext.databaseAccount.id}/containerconnections`
Make